pkgs/tools/misc/godu/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "godu";
+  version = "1.3.0";
+  rev = "v${version}";
```

Comment:
Only seems to be used by `fetchFromGitHub`. If so, this should just be an attribute of `fetchFromGitHub` to keep the builder's environment clean.